migrations/20220402211149-add-place-name-slug.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.addColumn(
+        'places',
+        'name_slug',
+        {
+          type: Sequelize.STRING,
+          allowNull: true,
+        },
+        { transaction },
+      );
+
+      const [places] = await queryInterface.sequelize.query(`select * from "places"`, { transaction });
+
+      for (const place of places) {
+        await queryInterface.bulkUpdate(
+          'places',
+          { name_slug: slugify(place.name) },
+          { id: place.id },
+          { transaction },
+        );
+      }
+    });
+  },
+
+  async down(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.removeColumn('places', 'name_slug', {
+        transaction,
+      });
+    });
+  },
+};
+
+function slugify(text) {
+  let normalizedText = text.toString().toLowerCase().trim();
+
+  const sets = [
+    { to: 'a', from: '[Ą]' },
+    { to: 'c', from: '[Ć]' },
+    { to: 'e', from: '[Ę]' },
+    { to: 'l', from: '[Ł]' },
+    { to: 'n', from: '[Ń]' },
+    { to: 'o', from: '[Ó]' },
+    { to: 's', from: '[Ś]' },
+    { to: 'z', from: '[ŹŻ]' },
+    { to: '-', from: "[·/_,:;']" },
```

Comment:
you could also escape () signs, it might occur in the name

Reply:
Every non-letter and non-digit character is removed later by regex.